README.md

```diff
@@ -71,6 +71,12 @@ It is also possible to view all test PDF files on the right side by opening:
 
 + http://localhost:8888/test/pdfs/?frame
 
+## Online Setup with a single click
+
+You can use [gitpod](https://gitpod.io) (A free online VS Code like IDE). With a single click it will launch a ready to code workspace with all the dependencies pre-installed and the web server running so that you can start hacking around straight away without wasting any precious time.
```

Comment:
I'd suggest prefacing this section by suggesting that people follow the `Getting the Code` section above, since that should still be considered the primary/recommended way to contribute.
Furthermore, let's remove "wasting time" bit, since that seems unnecessary and out of place.

All in all, I'd suggest something like the below (which also contains some fixes for weird casing):
```
It's recommended to follow [Getting the Code] above, however you can also use [Gitpod](https://gitpod.io) (a free online VS Code like IDE).
With a single click it will launch a ready to use workspace, with all the dependencies pre-installed and the web server running, so that you can start coding straight away.
```

Reply:
Thanks! Your recommendation here sounds more legit and simple to me.